### PR TITLE
Adds subroutes of admin/settings with navigation and UI of admin system settings

### LINKED ISSUE #183: Create all subroutes under admin/settings (with navigation) and UI for admin/settings/system only
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
image
Located at `admin/settings`
* Create all the sub routes under `admin/settings` with navigation.
* Create the UI for `admin/settings/system` (form validation may be handled in a separate issue)

Open separate issues for other routes under `admin/settings/`

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds subroutes of admin/settings with navigation and UI of admin system settings

#### Changes proposed in this pull request:
image

All commits are meaningful as well  important due to large number of files , please don't squash them.
Form Validation will be handled in a separate issue.

Fixes #183

### CODE REVIEW

**Comment by CosmicCoder96** [comment]
@niranjan94 @geekyd Deployment link : https://cc96-opev-frontend.herokuapp.com/admin/settings
Please review

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    {{ui-radio label=(t 'Google Storage') name='storage' value='google' onChange=(action (mut selectedStorage))}}
+    <span class="small text">
+      ({{t 'See'}}
+      <a href="https://cloud.google.com/storage/docs/migrating#defaultproj" target="_blank" rel="noopener nofollow">
+        {{t 'here'}}
+      </a>
+      {{t 'on how to obtain these keys.'}})
+    </span>
+  </div>
+  <div class="ui basic segment">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Remove segment

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      {{ui-radio label=(t 'Staging') name='environment' value='staging'}}
+    </div>
+    <div class="field">
+      {{ui-radio label=(t 'Production') name='environment' value='production'}}
+    </div>
+  </div>
+  <h3 class="ui header">
+    {{t 'App Name'}}
+  </h3>
+  <div class="grouped fields">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Remove this `<div class="grouped fields">` completely. The fields would still be displayed correctly. Not required for normal input fields. Generally used for checkboxes and radios.

Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Do the same for all other occurrences.

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    <label>
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Not label. Put this below the input with the `text muted` class.

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      {{t 'From Name'}}
+    </label>
+    {{input type='text' name='email_from_name'}}
+  </div>
+  <div class="ui hidden divider"></div>
+  <div class="field">
+    {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' current='smtp' onChange=(action (mut selectedMailOption))}}
+  </div>
+  <div class="ui basic segment">
+    <div class="grouped {{if (eq selectedMailOption 'sendgrid') 'disabled'}} fields">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Don't make fields _look_ disabled. Actually disable the inputs.

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+          <div class="default text">
+            {{t 'Choose Encryption type'}}
+          </div>
+          <i class="dropdown icon"></i>
+          <div class="menu">
+            <div class="item" data-value="tls">
+              {{t 'TLS'}}
+            </div>
+            <div class="item" data-value="tls-optional">
+              {{t 'TLS(optional)'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Add space. `TLS (optional)`

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs (hunk 4 above)
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: @niranjan94 Can yoi please check again, they are disabled. Thank you.

Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: They are disabled via css. Which is not right. If you query the input fields via js and check the disabled property, that still shows as enabled.

**Comment by CosmicCoder96** [comment]
@niranjan94 @geekyd Please review again

File: app/components/forms/admin/settings/system/storage-option.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+import { orderBy } from 'lodash';
+
+const { Component, computed } = Ember;
+
+export default Component.extend({
+  Regions: computed(function() {
+    return orderBy(amazonS3Regions);
+  }),
+
+  didInsertElement() {
```
Comment (niranjan94) [inline app/components/forms/admin/settings/system/storage-option.js]: Please don't do it like this ... Use ember's attribute bindings on the input `disabled` property to do it. We need elements to be reactive to the data.

File: app/components/forms/admin/settings/system/mail-settings.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+const { Component } = Ember;
+
+export default Component.extend({
+  didInsertElement() {
+    this.set('sendgrid', this.$('.sendgrid input'));
+    this.set('smtp', this.$('.smtp input, .smtp .dropdown'));
+  },
+  actions: {
+    toggleSetting(selected, unselected) {
+      var enable = this.get(selected);
```
Comment (niranjan94) [inline app/components/forms/admin/settings/system/mail-settings.js]: Same here

File: app/components/forms/admin/settings/system/storage-option.js (hunk 6 above)
Comment (CosmicCoder96) [inline app/components/forms/admin/settings/system/storage-option.js]: @niranjan94  You mean, `disabled = someAttributeBindingExpression` on each of the input fields explicitly?

Comment (niranjan94) [inline app/components/forms/admin/settings/system/storage-option.js]: Yes. Exactly

Comment (niranjan94) [inline app/components/forms/admin/settings/system/storage-option.js]: `{{input type='text' disabled=storageEnabled}}`

**Comment by CosmicCoder96** [comment]
@niranjan94 Please review again.

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+  {{ui-radio label=(t 'Amazon S3') name='storage' value='S3' onChange=(action (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-getting-started.html" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="grouped fields {{if (not-eq selectedStorage 'S3') 'disabled'}}">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: I have added grouped fields class here, just to ensure that fields are disabled via CSS as well to give the semantic-ui disabled look.

**Comment by CosmicCoder96** [comment]
Deployment link : https://cc96-opev-frontend.herokuapp.com/admin/settings

File: app/templates/admin/settings.hbs [hunk 9]
```diff
@@ -1 +1,31 @@
-{{outlet}}
+<div class="ui container grid">
```
Comment (niranjan94) [inline app/templates/admin/settings.hbs]: Remove `container` for now. Let this take the full width. We'll deal with this at the end

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+    {{input type='text' name='smtp_host' disabled=(not-eq selectedSetting 'smtp')}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Port'}}
+    </label>
+    {{input type='text' name='smtp_port'}}
+  </div>
+  <div class="less padding field">
+    <label>Encryption</label>
```
Comment (geekyd) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Add translation helper here.

**Comment by CosmicCoder96** [comment]
@niranjan94 I have removed the container, please review again.

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 11]
```diff
@@ -0,0 +1,76 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <h3 class="ui header">
+    {{t 'App Environment'}}
+    <div class="sub header">
+      {{t 'Where is the application running ?'}}
+    </div>
+  </h3>
+  <div class="field">
+    {{ui-radio label=(t 'Development') current='development' name='environment' value='development'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Required

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 12]
```diff
@@ -0,0 +1,76 @@
...
+    {{t 'App Name'}}
+    <div class="sub header">
+      {{t 'The name of your deployment. This will be used throughout this application.'}}
+    </div>
+  </h3>
+  <div class="field">
+    <label>
+      {{t 'App Name'}}
+    </label>
+    {{input type='text' name='app_name'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Required

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+    {{t 'App secret key'}}
+    <div class="sub header">
+      {{t 'Secret key for the application. Recommended is a random string minimum 20 characters in length.'}}
+    </div>
+  </h3>
+  <div class="field">
+    <label>
+      {{t 'Secret'}}
+    </label>
+    {{input type='text' name='secret_key'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Required, minlength: 16chars

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 14]
```diff
@@ -0,0 +1,76 @@
...
+    {{t 'Super Admin Email ID'}}
+    <div class="sub header">
+      {{t 'The email ID of the super admin user created during initial deployment.'}}
+    </div>
+  </h3>
+  <div class="field">
+    <label>
+      {{t 'Email'}}
+    </label>
+    {{input type='text' name='super_admin_email'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system-form.hbs]: Required, email

File: app/components/forms/admin/settings/system-form.js [hunk 15]
```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+const { Component } = Ember;
+
+export default Component.extend({
```
Comment (niranjan94) [inline app/components/forms/admin/settings/system-form.js]: Add form mixin to this and add the method to get validation rules

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' current='smtp' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="grouped {{if (not-eq selectedSetting 'smtp') 'disabled' }} fields">
+  <div class="field">
+    <label>
+      {{t 'Host'}}
+    </label>
+    {{input type='text' name='smtp_host' disabled=(not-eq selectedSetting 'smtp')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Required.

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Port'}}
+    </label>
+    {{input type='text' name='smtp_port'}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Required. Integer.

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 18]
```diff
@@ -0,0 +1,86 @@
...
+  </div>
+  <div class="less padding field">
+    <label>Encryption</label>
+    {{#ui-dropdown class='fluid selection' }}
+      {{input type='hidden' id='encryption_select' disabled=(not-eq selectedSetting 'smtp')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Required

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 19]
```diff
@@ -0,0 +1,86 @@
...
+          {{t 'No Encryption'}}
+        </div>
+      </div>
+    {{/ui-dropdown}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Username'}}
+    </label>
+    {{input type='text' name='smtp_username' disabled=(not-eq selectedSetting 'smtp')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Required

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 20]
```diff
@@ -0,0 +1,86 @@
...
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Password'}}
+    </label>
+    {{input type='password' name='smtp_password' disabled=(not-eq selectedSetting 'smtp')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Required

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+      {{t 'here'}}
+    </a>
+    {{t 'on how to get this token.'}})
+  </span>
+</div>
+<div class="sendgrid {{if (not-eq selectedSetting 'sendgrid') 'disabled'}} field">
+  <label>
+    {{t 'Sendgrid Token'}}
+  </label>
+  {{input type='text' name='sendgrid_token' disabled=(not-eq selectedSetting 'sendgrid')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Required

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 22]
```diff
@@ -0,0 +1,86 @@
...
+</div>
+<div class="field">
+  <label>
+    {{t 'From Name'}}
+  </label>
+  {{input type='text' name='email_from_name'}}
+</div>
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' current='smtp' onChange=(action (mut selectedSetting))}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Add another options. 'Disable email'

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 23]
```diff
@@ -0,0 +1,90 @@
...
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="grouped fields {{if (not-eq selectedStorage 'google') 'disabled'}}">
+  <div class="field">
+    <label>
+      {{t 'Bucket name'}}
+    </label>
+    {{input type='text' name='google_storage_bucket_name' disabled=(not-eq selectedStorage 'google')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 24]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Interoperable access key'}}
+    </label>
+    {{input type='text' name='google_storage_access_key' disabled=(not-eq selectedStorage 'google')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 25]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Interoperable access secret'}}
+    </label>
+    {{input type='text' name='google_storage_access_secret' disabled=(not-eq selectedStorage 'google')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 26]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+</div>
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'Amazon S3') name='storage' value='S3' onChange=(action (mut selectedStorage))}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

**Comment by CosmicCoder96** [comment]
@niranjan94  @geekyd Please review again.

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 27]
```diff
@@ -0,0 +1,90 @@
...
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="grouped fields {{if (not-eq selectedStorage 'S3') 'disabled'}}">
+  <div class="less padding field">
+    <label>Region</label>
+    {{#ui-dropdown class='fluid selection'}}
+      {{input type='hidden' id='amazon_s3_region' disabled=(not-eq selectedStorage 'S3')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 28]
```diff
@@ -0,0 +1,90 @@
...
+          </div>
+        {{/each}}
+      </div>
+    {{/ui-dropdown}}
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Bucket name'}}
+    </label>
+    {{input type='text' name='amazon_s3_bucket_name' disabled=(not-eq selectedStorage 'S3')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 29]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Key'}}
+    </label>
+    {{input type='text' name='amazon_s3_key' disabled=(not-eq selectedStorage 'S3')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 30]
```diff
@@ -0,0 +1,90 @@
...
+  </div>
+  <div class="field">
+    <label>
+      {{t 'Secret'}}
+    </label>
+    {{input type='text' name='amazon_s3_secret' disabled=(not-eq selectedStorage 'S3')}}
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Required

**Comment by niranjan94** [comment]
@CosmicCoder96 good suggestion by @geekyd . Please do that. And that way you don't have to bother with disabling the fields either.

File: app/components/forms/admin/settings/system-form.js (hunk 15 above)
Comment (CosmicCoder96) [inline app/components/forms/admin/settings/system-form.js]: @niranjan94 I had mentioned in the PR that form validation will be handled in a separate issue, as this was taking some time, I will add it here itself then.

**Comment by CosmicCoder96** [comment]
@niranjan94 I had mentioned in the PR that form validation will be handled in a separate issue, as this was taking more than a days time, I will add it here itself then.

**Comment by CosmicCoder96** [comment]
@niranjan94 Please Review. I will create a separate issue for form validation.

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 31]
```diff
@@ -0,0 +1,89 @@
...
+  {{input type='text' name='email_from_name'}}
+</div>
+<div class="ui hidden divider"></div>
+<div class="field">
+  {{ui-radio label=(t 'Disable email') name='mail_setting' value='disable' current='disable' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="field">
+  {{ui-radio label=(t 'SMTP') name='mail_setting' value='smtp' onChange=(action (mut selectedSetting))}}
+</div>
+<div class="{{if (not-eq selectedSetting 'smtp') 'system hidden'}}">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Don't use a hidden class. Use a full if statement and don't render the things not required.

File: app/templates/components/forms/admin/settings/system/mail-settings.hbs [hunk 32]
```diff
@@ -0,0 +1,89 @@
...
+  {{ui-radio label=(t 'Sendgrid API') name='mail_setting' value='sendgrid' onChange=(action (mut selectedSetting))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="https://sendgrid.com/docs/User_Guide/Settings/api_keys.html" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to get this token.'}})
+  </span>
+</div>
+<div class="{{if (not-eq selectedSetting 'sendgrid') 'system hidden'}} field">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/mail-settings.hbs]: Don't use a hidden class. Use a full if statement and don't render the things not required.

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 33]
```diff
@@ -0,0 +1,90 @@
...
+  {{ui-radio label=(t 'Google Storage') name='storage' value='google' onChange=(action  (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="https://cloud.google.com/storage/docs/migrating#defaultproj" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="{{if (not-eq selectedStorage 'google') 'system hidden'}}">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Don't use a hidden class. Use a full if statement and don't render the things not required.

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 34]
```diff
@@ -0,0 +1,90 @@
...
+  {{ui-radio label=(t 'Amazon S3') name='storage' value='S3' onChange=(action (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'See'}}
+    <a href="http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-getting-started.html" target="_blank" rel="noopener nofollow">
+      {{t 'here'}}
+    </a>
+    {{t 'on how to obtain these keys.'}})
+  </span>
+</div>
+<div class="{{if (not-eq selectedStorage 'S3') 'system hidden'}}">
```
Comment (niranjan94) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Don't use a hidden class. Use a full if statement and don't render the things not required.

**Comment by CosmicCoder96** [comment]
@niranjan94 @geekyd  Removed the hidden class, please review

File: app/templates/components/forms/admin/settings/system/storage-option.hbs [hunk 35]
```diff
@@ -0,0 +1,92 @@
+<h3 class="ui header">
+  {{t 'Storage'}}
+  <div class="sub header">
+    {{t 'Where to store data ?'}}
+  </div>
+</h3>
+<div class="field">
+  {{ui-radio label=(t 'Local') current ='local' name='storage' value='local' onChange=(action (mut selectedStorage))}}
+  <span class="small text">
+    ({{t 'stored in'}} /static/media/folder)
```
Comment (geekyd) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: Please add translation helper here.

Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/system/storage-option.hbs]: @geekyd That's a partial URL, I left it out intentionally.

**Pull request merged**